app/assets/src/components/samples.scss

```diff
@@ -0,0 +1,58 @@
+@import "~styles/themes/colors";
+@import "~styles/typography";
+
+/* BackgroundModal */
+.sampleList {
```

Comment:
Did the recommended notation for selectors changed because we are using css modules?

Reply:
Yeah. If you do `.sample-list`, in JSX you'd have to do `className={cs['sample-list']}` instead of `{cs.sampleList}`.

Reply:
Ok, i see you already updated the development guid.